app/response/utils.py

```diff
@@ -574,6 +629,44 @@ def send_note_email(request_id, note_content, privacy, email_content, **kwargs):
                                   note_content=note_content)
 
 
+def _send_instruction_email(request_id, instruction_content, privacy, email_content, **kwargs):
+    """
+    Function that sends email detailing a instruction has been added to a request.
+    Send email to the requester and bcc all agency users detailing an instruction has been added to the request as a
+    response.
+
+    :param request_id: FOIL request ID for the specific instruction
+    :param instruction_content: string content of the instruction
+    :param privacy: privacy option of the instruction
+    :param email_content: string of HTML email content to be created and stored as a email object
+
+    :return:
+    """
+    subject = 'Response Added'
+    agency_name = Requests.query.filter_by(id=request_id).first().agency.name
+    bcc = get_agencies_emails(request_id)
+    requester_email = UserRequests.query.filter_by(request_id=request_id,
+                                                   request_user_type=REQUESTER).first().user.email
+    # Send email with files to requester and bcc agency_ein users as privacy option is release
+    if privacy == PRIVATE:
+        email_content = render_template(kwargs['email_template'],
+                                        request_id=request_id,
+                                        agency_name=agency_name,
+                                        instruction_content=instruction_content)
+        safely_send_and_add_email(request_id,
+                                  email_content,
+                                  subject,
+                                  bcc=bcc)
+    else:
+        to = [requester_email]
+        safely_send_and_add_email(request_id,
+                                  email_content,
+                                  subject,
+                                  to=to,
```

Comment:
Changing to `to=[requester_email]` and removing the assignment above this call should work just fine and will maintain readability.